src/modules/PixelRatio.js

```diff
@@ -0,0 +1,38 @@
+const ReactPrimitives = require('../ReactPrimitives');
+
+const PixelRatio = {
```

Comment:
It seems like `PixelRatio` can be implemented purely as a function of `Dimensions` in most cases, so i'm leaving that as an option for targets. i might change this to override it if it's provided though